app/controllers/dashboard.js

```diff
@@ -22,6 +22,10 @@ export default class DashboardController extends Controller {
         loading: false
     };
 
+    get showTopMembers() {
+        return this.feature.get('emailAnalytics');
```

Comment:
This always returns true, even if I switch the setting off in email newsletter settings :thinking: